.github/workflows/update-containerd.yml

```diff
@@ -0,0 +1,112 @@
+name: Update containerd
+on:
+  pull_request:
+    branches:
+      - master
+  workflow_dispatch: {}
+
+jobs: 
+  build-and-deploy:
+    runs-on: ubuntu-latest
+    environment: test
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-go@v2
+        with:
+          go-version: '^1.16'
+      - uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+      - name: Cache cargo registry
+        uses: actions/cache@v2
+        id: cache-cargo
+        continue-on-error: false
+        with:
+          path: |
+            ~/.cargo
+            ~/.cargo/bin/rg 
+            ~/.cargo/bin/delta   
+          key: ${{ runner.os }}-rg 
+      - name: Cache sponge
+        uses: actions/cache@v2
+        id: cache-sponge
+        continue-on-error: false
+        with:
+          path: |
+            ~/.cargo/bin/sponge   
+          key: ${{ runner.os }}-sponge 
+      - run: cargo install ripgrep git-delta
+        if: steps.cache-cargo.outputs.cache-hit != 'true'
+      - run: sudo apt update && sudo apt install -y moreutils && sudo cp /usr/bin/sponge ~/.cargo/bin/sponge  
+        if: steps.cache-sponge.outputs.cache-hit != 'true'
+      - name: update containerd
+        id: versions
+        run: |
+          set -euox pipefail
+
+          which rg
+
+          work=$(mktemp -d)
+          pushd $work
+          trap 'popd; rm -rf $work' EXIT TERM
+
+          git clone https://github.com/containerd/containerd
+          cd containerd/
+          
+          latest_two_major_minor="$(git tag -l | sort -V | rg 'v[0-9]+.[0-9]+.[0-9]+$' | cut -d'.' -f1,2 | uniq | tail -n 2)"
```

Comment:
Clever! It’s nice that you are using git tags in the repo to determine the latest versions